src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -255,7 +255,7 @@
   <data name="Open" xml:space="preserve">
     <value>Open</value>
   </data>
-  <data name="OpenInNewTab" xml:space="preserve">
+  <data name="OpenDirectoryInNewTab" xml:space="preserve">
```

Comment:
Can you update this in the other resource files so that it doesn't reset the translation?

Reply:
Fixed